src/tools/management/commands/process_summaries.py

```diff
@@ -0,0 +1,58 @@
+import os
+from concurrent.futures import ThreadPoolExecutor
+from time import sleep
+
+from django.core.management.base import BaseCommand
+from django.db import connection
+
+from api.models import SummaryCacheQueue
+from api.utils.summary_cache import update_summary_cache
+
+
+class Command(BaseCommand):
+    LOCK_ID = 8080
+    WAIT_SECONDS = 5
+
+    def acquire_lock(self):
+        with connection.cursor() as cursor:
+            cursor.execute("SELECT pg_try_advisory_lock(%s);", [self.LOCK_ID])
```

Comment:
What happens if a lock can't be acquired for some reason? If a lock already exists, I guess this returns `None`; but what if an exception is raised, or some other result happens?

Reply:
pg_try_advisory_lock returns back true or false

I've wrapped both acquire and release in try/except to log if there's some DB exception